src/main/java/sirius/db/es/Elastic.java

```diff
@@ -702,6 +702,23 @@ public <E extends ElasticEntity> ElasticQuery<E> select(Class<E> type) {
         return new ElasticQuery<>(mixing.getDescriptor(type), getLowLevelClient());
     }
 
+    /**
+     * Creates a new query which can be used to fetch entities of the given types.
+     * <p>
+     * Note: All entities to query across must extend a common super class.
+     *
+     * @param commonSuperClass the common super class of all entities to query across
+     * @param types            the types of the entities to query across
+     * @param <E>              the generic common type of the entities to query across
+     * @return a new query which can be used to fetch entities of the given types
+     */
+    @SafeVarargs
+    public final <E extends ElasticEntity> ElasticQuery<E> selectMultiple(Class<E> commonSuperClass,
+                                                                          Class<? extends E>... types) {
+        return new ElasticQuery<E>(mixing.getDescriptor(types[0]),
```

Comment:
Why not block `ElasticEntity` itself as concrete super-class here if it may lead to issues?